src/core/operations/Arithmetic.js

```diff
@@ -0,0 +1,208 @@
+import Utils from "../Utils.js";
+
+/**
+ * Math operations on numbers.
+ *
+ * @author bwhitn [brian.m.whitney@outlook.com]
+ * @copyright Crown Copyright 2016
+ * @license Apache-2.0
+ *
+ * @namespace
+ */
+const Arithmetic = {
+
+    /**
+     * @constant
+     * @default
+     */
+    DELIM_OPTIONS: ["Space", "Comma", "Semi-colon", "Colon", "Line feed", "CRLF"],
+
+    /**
+     * @constant
+     * @default
+     */
+    OPERATIONS: [
+        "Sum",
+        "Sub",
+        "Multiply",
+        "Divide",
+        "Mean",
+        "Median",
+        "Standard Deviation"
+    ],
+
+
+    /**
+     *
+     *
+     * @param {string} input
+     * @param {Object[]} args
+     * @returns {number}
+     */
+    computeSum: function(input, args) {
```

Comment:
It's convention to name operation functions starting with the word "run", so this would be `runSum` and the next would be `runSub` etc.